HIP/hip-906.md

```diff
@@ -79,14 +77,14 @@ contract Assembly {
 
 > Note: Just as with the Hedera Token Service system contract, the Hedara Account Service system contract will not be callable using `delegatecall` from a contract. 
 
-The following table describes the function selector for the new `hbarAllownace` and `hbarApprove` functions and the associated function signature and response.
+The following table describes the function selector for the new `hbarAllownace` and `hbarApprove` functions and the associated function signature and response in `IHRC632`.
 
 | Function Selector Hash | Function Signature                             | Response                                                                               | 
 |------------------------|------------------------------------------------|----------------------------------------------------------------------------------------|
 | 0xbbee989e             | hbarAllowance(address spender)                 | (ResponseCode, uint256 - amount of hbar allowances currently available to the spender) | 
```

Comment:
This formatting of the response - with the "comment" sort of inline - is that the way it's done?